docs/source/conf.py

```diff
@@ -69,11 +69,18 @@
     'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
     'pandas': ('https://pandas.pydata.org/pandas-docs/stable/', None),
     'qtutils': ('https://qtutils.readthedocs.io/en/stable/', None),
-    'pyqtgraph': ('https://pyqtgraph.readthedocs.io/en/latest/', None), # change to stable once v0.11 is published
+    'pyqtgraph': (
+        'https://pyqtgraph.readthedocs.io/en/latest/',
+        None,
+    ),  # change to stable once v0.11 is published
     'matplotlib': ('https://matplotlib.org/', None),
     'h5py': ('http://docs.h5py.org/en/stable/', None),
     'pydaqmx': ('https://pythonhosted.org/PyDAQmx/', None),
-    'qt': ('', 'pyqt5-modified-objects.inv') # from https://github.com/MSLNZ/msl-qt/blob/master/docs/create_pyqt_objects.py under MIT License
+    'qt': (
```

Comment:
Is this formatting due to running it through `black`? I have been meaning to advocate for a longer line length if we decide to format with `black` - probably 120 chars?

Reply:
New code formatted with `black` has already been added elsewhere, and uses the default (`--line-length 88`), which I'd be keen to stick with.